test/rss.cpp

```diff
@@ -489,6 +491,48 @@ TEST_CASE("RssFeed::sort() correctly sorts articles", "[rss]")
 		REQUIRE(articles[3]->pubDate_timestamp() == 23);
 		REQUIRE(articles[4]->pubDate_timestamp() == 7);
 	}
+
+	SECTION("random")
+	{
+		ArticleSortStrategy ss;
+		ss.sm = ArtSortMethod::RANDOM;
+
+		class G
+		{
+			private:
+				uint32_t i;
+			public:
+				typedef uint32_t result_type;
+
+				result_type min()
```

Comment:
This and `max()` need to be `static`; MacOS [tries to call `G::min()`](https://travis-ci.org/newsboat/newsboat/jobs/500580116#L2623) and fails because this implementation of `min()` needs an object (i.e. `G::min()` is invalid, only `rng.min()` is).